src/cljs/owlet_ui/components/search_bar.cljs

```diff
@@ -8,41 +8,72 @@
 
 (defonce suggestion-count (reagent/atom 16))
 
+(defonce this-scroll (atom 0))
+
+(defonce last-scroll (atom 0))
+
 (defn toggle-suggestions []
   (if-let [suggestions (aget (js->clj (js/document.getElementsByClassName "rc-typeahead-suggestions-container")) 0)]
     (let [hidden (.-hidden suggestions)]
       (when-not (nil? suggestions)
         (set! (.-hidden suggestions) (not hidden))))))
 
+(defn show-search []
+  (let [search (aget (js->clj (js/document.getElementsByClassName "form-control")) 0)]
+    (set! (-> search .-style .-height) "50px")
```

Comment:
@dviramontes We could do something similar to what we did with the sidebar, using an atom for the classes and then swapping them and having CSS to modify